_integration-schemas/ga360/ga_sessions.md

```diff
@@ -0,0 +1,280 @@
+---
+tap: "ga360"
+version: "1"
+key: "ga360"
+
+name: "ga_sessions"
+doc-link: "https://support.google.com/analytics/answer/3437719?hl=en&ref_topic=3416089"
+singer-schema: "https://github.com/singer-io/tap-ga360/blob/master/tap_ga360/schemas/ga_sessions.json"
+description: "This table contains information about general Google Analytics sessions."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "BigQuery Export schema"
+    doc-link: "https://support.google.com/analytics/answer/3437719?hl=en&ref_topic=3416089"
+
+attributes:
+  - name: "fullVisitorId"
+    type: "string"
+    primary-key: true
+    description: "The unique visitor ID - also known as client ID."
+    foreign-key-id: "full-visitor-id"
+
+  - name: "visitId"
+    type: "number"
+    primary-key: true
+    description: "An identifier for this session."
+    foreign-key-id: "visit-id" 
+
+  - name: "channelGrouping"
+    type: "string"
+    description: "The Default Channel Group associated with an end user's session for this View."
+
+  - name: "clientId"
+    type: "string"
+    description: "Unhashed version of the Client ID for a given user associated with any given visit/session."
+
+  - name: "customDimensions"
+    type: "array"
+    description: "This section contains any user-level or session-level custom dimensions that are set for a session. This is a repeated field and has an entry for each dimension that is set."
+    subattributes:
+      - name: "index"
+        type: "number"
+        description: "The index of the custom dimension."
+      - name: "value"
+        type: "string"
+        description: "The value of the custom dimension."
+
+  - name: "date"
```

Comment:
This is the table's replication key - it looks like it is an actual field!

Bump this up after all the primary key attributes and add the `replication-key: true` flag. It's probably a good idea to also include a link to the replication section you're gonna write in the description for this attribute.